nautobot/dcim/models/device_component_templates.py

```diff
@@ -250,7 +250,7 @@ class Meta:
 
     def instantiate(self, device):
         try:
-            status = Status.objects.get_for_model(Interface).get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
+            status = Status.objects.get_for_model(Interface).get(name="Active")
```

Comment:
Bleh, I'd forgotten that we had this fallback. We should really allow/require InterfaceTemplate objects to define a `default/initial_status` parameter and remove this fallback.

Reply:
Filed #3474.